src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -281,19 +285,19 @@ public ContextAction(StandardActions command, LinkedFileViewModel linkedFile, Pr
                                 () -> !linkedFile.getFile().isOnlineLink()
                                         && linkedFile.getFile().findIn(databaseContext, preferencesService.getFilePreferences()).isPresent()
                                         && !linkedFile.isGeneratedNameSameAsOriginal(),
-                                linkedFile.getFile().linkProperty());
+                                linkedFile.getFile().linkProperty(), stateManager.getSelectedEntries().get(0).getFieldsObservable());
```

Comment:
The editor is notified in 
https://github.com/JabRef/jabref/blob/d190268813e7e65b09a744158fcfa65371f5a28d/src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java#L210
about the entry to show. So that's probably a better way to get the entry instead of pushing down the state manager.

Reply:
bindToEntry does directly bind the linkedFileViewModel right now, but I'll see what I can do.